catalog/data-relay/manifest.yaml

```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../../schema.json
+---
+version: 2.16.1
+name: data-relay
```

Comment:
This is the name of the app shown in the store, feel free to use `Data Relay (Beta)` here.